screens/Scan/SharingStatusModal.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+import {Theme} from '../../components/ui/styleUtils';
+import {Modal} from '../../components/ui/Modal';
+import {Pressable, Dimensions} from 'react-native';
+import {Button, Column, Row, Text} from '../../components/ui';
+import testIDProps from '../../shared/commonUtil';
+import {Icon} from 'react-native-elements';
+
+export const SharingStatusModal: React.FC<SharingStatusModalProps> = props => {
+  const {t} = useTranslation('ScanScreen');
+
+  return (
+    <React.Fragment>
+      <Modal
+        isVisible={props.isVisible}
+        showClose={false}
+        {...testIDProps(props.testId)}>
+        <Column
+          margin="64 0"
+          crossAlign="center"
+          style={Theme.SelectVcOverlayStyles.sharedSuccessfully}>
+          {props.image}
+          <Text
+            testID="sharingStatusTitle"
+            margin="20 0"
+            style={Theme.TextStyles.bold}
+            size={'large'}>
+            {props.title}
+          </Text>
+          <Text
+            testID="sharingStatusMessage"
+            align="center"
+            margin="0 33 0 33"
+            style={Theme.TextStyles.regular}
+            color={Theme.Colors.statusMessage}>
+            {props.message}
+          </Text>
+        </Column>
+        {props.status === 'successfullyShared' ? (
+          <Row
+            align="space-evenly"
+            style={{marginBottom: Dimensions.get('screen').height * 0.06}}>
+            <Pressable testID="successfulVcSharedHomeIcon">
+              <Icon
+                name="home"
```

Comment:
Suggestion: We already have home svg image in svg.tsx, then we can reuse that in here as well instead of icon

Reply:
We are using SVGs in place of PNG's when there is no chance with the ICON component.

Reply:
Since we've already created a component for an element. Using that will ensure consistency across all components, right?

Reply:
I have followed that. However, I couldn't able to add styles for SVG's separately. Attached screenshot.
[success screen](https://github.com/mosip/inji/assets/106086523/bb2040bd-364a-4397-b754-5c1979a2d55c)

Reply:
Respective Screenshot [success screen](https://github.com/mosip/inji/assets/106086523/b7953ec7-0866-4fb2-95e4-386de436ebdd)

Reply:
I tried however we can't able to implement styles for the SVG home to re-use it.